packages/grid/x-data-grid/src/components/base/GridBody.tsx

```diff
@@ -175,6 +176,8 @@ function GridBody(props: GridBodyProps) {
           disableVirtualization={isVirtualizationDisabled}
         />
       )}
+
```

Comment:
```suggestion
```

Reply:
Weird, I remember Prettier adding it back, but it's not the case anymore 😅

Reply:
Here we go again, `yarn proptypes` adds it back 😀

<img width="592" src="https://user-images.githubusercontent.com/13808724/235910554-2575eda3-2bea-4263-bc6e-457dcd396578.png">